packages/core/src/data-access/commits.ts

```diff
@@ -1,11 +1,11 @@
-import { Commit } from '$core/browser'
-import { database } from '$core/client'
-import { HEAD_COMMIT } from '$core/constants'
-import { Result, TypedResult } from '$core/lib'
-import { LatitudeError, NotFoundError } from '$core/lib/errors'
-import { commits } from '$core/schema'
 import { and, desc, eq, isNotNull } from 'drizzle-orm'
 
+import { Commit, HEAD_COMMIT } from '../browser'
+import { database } from '../client'
+import { LatitudeError, NotFoundError } from '../lib/errors'
+import { Result, TypedResult } from '../lib/Result'
+import { commits } from '../schema'
```

Comment:
the tradeoff is that we use relative imports in all [just in time packages](https://turbo.build/repo/docs/core-concepts/internal-packages#just-in-time-packages) (those that aren't compiled). I tried to use [node's subpath imports](https://turbo.build/repo/docs/guides/tools/typescript#use-nodejs-subpath-imports-instead-of-typescript-compiler-paths), as recommended by turborepo, but nextjs doesn't support them. 

We can review this when nextjs gets its shit together.

Reply:
buff